hack/assets/assets.go

```diff
@@ -144,14 +144,16 @@ func GetImageList(latest bool, optional bool, kubernetesVersion string, groups [
 
 // GetVclusterImages returns a list of vcluster images
 func GetVclusterImages(latest, optional bool, cleanTag string) []string {
-	images := []string{"ghcr.io/loft-sh/vcluster-oss:" + cleanTag}
+	images := []string{
+		"ghcr.io/loft-sh/vcluster-oss:" + cleanTag,
+		config.DefaultHostsRewriteImage,
+	}
 	if !optional {
 		if latest {
 			images = nil
 		}
 		images = append(images,
 			"ghcr.io/loft-sh/vcluster-pro:"+cleanTag,
-			config.DefaultHostsRewriteImage,
 		)
 	}
 	return images
```

Comment:
Unrelated to this change, but this functional also seems quite hard to read:
```
    images := []string{...}
	if !optional {
		if latest {
			images = nil
```
Actually a better question might be, what's the point of the latest flag at all ? Didn't seem to exist prior to 0.25. Having both a latest and an optional flag makes more complicated scenarios. Could we get rid of latest, or have the flags be mutually exclusive and latest is the default ? Then this function would just be: 
```
	if optional {
        return []string{ "ghcr.io/loft-sh/vcluster-oss:" + cleanTag, config.DefaultHostsRewriteImage,}
	} 
	return []string{"ghcr.io/loft-sh/vcluster-pro:"+cleanTag}
```
Or we could have both be able to be set, and append if optional is true. Either way it seems simpler that way.